mlir/lib/Target/LLVMIR/Dialect/NVVM/NVVMToLLVMIRTranslation.cpp

```diff
@@ -227,14 +228,14 @@ class NVVMDialectLLVMIRTranslationInterface
     } else if (attribute.getName() ==
```

Comment:
What about the other attributes above? Do we plan to change them as well? If yes, it'd be better to do everything at once.

Reply:
Yes, I plan to replace all !nvvm.annotations with attributes. This change is already fairly large and I would prefer to avoid a single monolithic PR to make debugging any issues easier and to prevent unnecessary churn if it needs to be reverted. Would it be alright to address these now and the others in separate follow ups?

Reply:
Sure sounds great